tony-history-server/startTHS

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env sh
+###########################################################################################################
+# NAME: startTHS
+#
+# DESCRIPTION:
+# This is a modified version of the `playBinary` script that was generated after `gradle dist`.
+#
+#
+# INPUT: JVM options (-Dhttp.port, -Dhttp.address, -Dconfig.resource, etc.)
+#
+#
+# OUTPUT: Logs of play.core.server.ProdServerStart
+#
+#
+# ENVIRONMENT VARIABLES:
+# JAVA_HOME: location of Java executable
+#
+#
+# NOTES: This script should only be run when it's in the distribution zip/tar.
+# Examples: after `gradle dist` from root, unzip the zip/tar in $root/build/distributions, then run:
+# ```
+# $ $root/build/distributions/startTHS -Dhttp.port=8080 -Dconfig.resource=prod.conf
+# ```
+# This will set the port to 8080, and Play will use `conf/prod.conf` instead of the default
+# `application.conf`. For more details, please see
+#
+#
+# EXIT CODE: Same as default playBinary script
+#
+#
+# CHANGELOG:
+# OCT 30 2018 PHAT TRAN
+############################################################################################################
+PRG="$0"
+
+# Need this for relative symlinks.
+while [ -h "$PRG" ] ; do
+    ls=`ls -ld "$PRG"`
+    link=`expr "$ls" : '.*-> \(.*\)$'`
+    if expr "$link" : '/.*' > /dev/null; then
+        PRG="$link"
+    else
+        PRG=`dirname "$PRG"`"/$link"
+    fi
+done
+SAVED="`pwd`"
+cd "`dirname \"$PRG\"`/.." >/dev/null
+APP_HOME="`pwd -P`"
+cd "$SAVED" >/dev/null
+
+APP_NAME="playBinary"
```

Comment:
Is this still accurate? Should this be `tony-history-server`?

Reply:
Yeah, it should be `tony-history-server`.

Reply:
Fixed: https://github.com/linkedin/TonY/pull/62/commits/b5c273129ed921f6137fe822ea156375e845974b